inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/api/WorkloadConst.java

```diff
@@ -0,0 +1,16 @@
+package de.tudarmstadt.ukp.inception.workload.dynamic.api;
+
+public class WorkloadConst
```

Comment:
I still think that the default constants should not be in the dynamic workload module. @reckart  What do you think?

Reply:
Ahh ok, I thought you just wanted them to be more on "top" instead of inside the "manager" package, but yes maybe something similar to what it is in webanno? (de.tudarmstadt.ukp.clarin.webanno.model)